server/src/main/java/org/elasticsearch/search/aggregations/support/AggregationContext.java

```diff
@@ -303,6 +313,7 @@ public final AggregationUsageService getUsageService() {
         private final Supplier<Boolean> isCancelled;
         private final Function<Query, Query> filterQuery;
         private final boolean enableRewriteToFilterByFilter;
+        private final boolean isInOrderExecutionRequired;
```

Comment:
nit: I think the general naming convention would be to call this boolean just `inOrderExecutionRequired` but keep the leading `is` in the getter?

Reply:
And the method now has the word "sort" in it.